WireDomain/Sources/WireDomain/OneOnOneResolver.swift

```diff
@@ -121,7 +121,7 @@ struct OneOnOneResolver: OneOnOneResolverProtocol {
 
         /// Sync the user MLS conversation from backend.
         let mlsGroupID = try await conversationsRepository.pullMLSOneToOneConversation(
-            userID: userID.uuid.uuidString,
+            for: userID.uuid.uuidString,
```

Comment:
Isn't `userID` better in this case since the argument is `String`?

Reply:
fixed in https://github.com/wireapp/wire-ios/pull/2091/commits/cbebb8686ef60bc47af686531cbc8732218095fe thx